functions/src/test/mocks.ts

```diff
@@ -239,7 +239,7 @@ export const firestoreMock = {
             return null
```

Comment:
(if you expand the code above this, on line 236, you'll see where we check the collection ID. This lets us return specific data depending on which collection we're looking at. This would have to be expanded to work with the new format.)